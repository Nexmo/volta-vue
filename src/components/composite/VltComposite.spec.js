import { shallowMount } from '@vue/test-utils';
import { VltComposite } from '..';

describe('vlt-composite', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(VltComposite);
  });

  test('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
