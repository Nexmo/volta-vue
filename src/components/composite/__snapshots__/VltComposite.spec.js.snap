// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vlt-composite renders correctly appended button 1`] = `
<div
  class="Vlt-composite"
>
  <!--v-if-->
  
  <vlt-input-stub
    class=""
    disabled="false"
    size="20"
    textarea="false"
    type="text"
  />
  
  <!--v-if-->
</div>
`;

exports[`vlt-composite renders correctly prended button 1`] = `
<div
  class="Vlt-composite"
>
  <!--v-if-->
  
  <vlt-input-stub
    class=""
    disabled="false"
    size="20"
    textarea="false"
    type="text"
  />
  
  <!--v-if-->
</div>
`;
